ci/ci/ci.py

```diff
@@ -733,6 +739,15 @@ async def on_startup(app):
 '''
     )
 
+    async with await request_retry_transient_errors(
+        app['client_session'],
+        'GET',
+        deploy_config.url('auth', '/api/v1alpha/users'),
+        headers=service_auth_headers(deploy_config, 'auth'),
+    ) as resp:
+        users = await resp.json()
+        app['developers'] = [u for u in users if u['is_developer'] == 1 and u['username'] != 'test-dev']
```

Comment:
worth all of us remembering that the developers only get updated when CI restarts.